examples/managed_instance_group/main.tf

```diff
@@ -19,10 +19,18 @@ locals {
     "130.211.0.0/22",
     "35.191.0.0/16",
   ]
+  target_tags                    = [
+    "container-vm-test-mig"]
```

Comment:
The trailing `]` should be on a separate line here

Reply:
@kopachevsky @Jberlinsky 
[Fixed](https://github.com/terraform-google-modules/terraform-google-container-vm/compare/21506b0e8d88a8bee72d02dc63094b8276ccd288..b6d0f962d7a2074d4392d31a3368b8c6808df77b)